netbox_librenms_plugin/templates/netbox_librenms_plugin/htmx/create_platform_modal.html

```
@@ -13,10 +13,19 @@
 {#   server_key              - (optional) LibreNMS server key, forwarded as hidden field #}
 <div class="modal-header">
     <h5 class="modal-title" id="createPlatformModalLabel">
-        <i class="mdi mdi-alert"></i> Create New Platform
+        {% if libre_device %}<i class="mdi mdi-cog-outline"></i> Manage platform — <code>{{ librenms_os }}</code>{% else %}<i class="mdi mdi-alert"></i> Create New Platform{% endif %}
     </h5>
     <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
 </div>
+{% if libre_device %}
+{# Map the LibreNMS OS to an EXISTING platform (separate form -> add_platform_mapping). #}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)